ooni/nettest.py

```diff
@@ -730,6 +761,8 @@ class NetTestCase(object):
 
     localOptions = {}
 
+    _totalInputs = -1
```

Comment:
Yes you are right. It was meant to be some sort of flag, but I realise now that I actually don't check it. I think at this point I should probably use None and check for None in the above.